Readme.md

```diff
@@ -125,9 +125,22 @@ Custom HTML file:
 ```html
 $ echo \
   '<!doctype html>
-  <link rel="stylesheet" href="/style.css" />
-  <script src="/elm.js"></script>
-  ' > index.html
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Elm Live Demo</title>
```

Comment:
Let’s keep the example minimal and keep the title and charset out. I’d use them in any real page (as well as a bunch of other things), but each line in the example makes the whole thing feel less simple.